src/Notepad.php

```diff
@@ -416,4 +416,46 @@ public static function showForItem(CommonDBTM $item, $withtemplate = 0)
         }
         return true;
     }
+
+    public static function showNotes(CommonDBTM $item)
```

Comment:
I guess it could be possible to replace this whole method by a template file that could be called like this:
```
{{ include('components/notepad/notes.html.twig', {'item': item}, with_context = false) }}
```